src/patternfly/demos/Banner/__banner-page-main-section-content.hbs

```diff
@@ -0,0 +1,4 @@
+{{#> content}}
```

Comment:
Same as above, I think you can drop `-content` in the partial name or use `-demo`...